### DISCUSSION TITLE
Profiling not working when I build from the latest source.

### DISCUSSION THREAD

**Discussion by ajgillette** [coroot/coroot#166] [Q&A] [answered]
I keep getting an error => 
![image003](https://github.com/coroot/coroot/assets/5501726/a4761e45-5650-4161-8e30-9e630dcc1808)

clickhouse error: code : 60, message: Table default.profiling_profiles doesn't exist

**Comment by ajgillette** [comment]
Is there some package other than coroot that needs to be installed?

**Comment by apetruhin** [comment, marked as answer]
Please install the [coroot-cluster-agent](https://coroot.com/docs/coroot-community-edition/profiling/overview), responsible for collecting and storing profiling data in ClickHouse. It will create the necessary tables and views.
Using Helm is the easiest way to [install](https://coroot.com/docs/coroot-community-edition/getting-started/installation) Coroot along with all its components.
